reactor-core/src/test/java/reactor/core/publisher/BlockingTests.java

```diff
@@ -26,29 +26,30 @@
 import java.util.function.Function;
 import java.util.stream.Stream;
 
-import org.junit.AfterClass;
+import org.junit.jupiter.api.AfterAll;
 import org.junit.Assert;
-import org.junit.BeforeClass;
-import org.junit.Test;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
 import reactor.core.Exceptions;
 import reactor.core.scheduler.Scheduler;
 import reactor.core.scheduler.Schedulers;
 import reactor.test.StepVerifier;
 
 import static org.assertj.core.api.AssertionsForClassTypes.*;
+import static org.junit.jupiter.api.Assertions.assertThrows;
```

Comment:
IMO we should go with AssertJ and not introduce an intermediate step that we will need to change later

Reply:
Well, those 250+ manual changes are done now, so if that's a stepping stone for automatic migration to assertJ, I'd rather use them :).

Reply:
Although I am sorry that you spent time on it, I think "already done manually" isn't a good argument in favour of introducing a "legacy"

I see no point in merging the use of `org.junit.jupiter.api.Assertions.assertThrows` while we could go with AssertJ from the beginning.

Reply:
As I said, I hope it's easier to migrate (via plugin or dumb search/replace) from `Assertions.assertThrows` to assertJ than from `@Test(expected = Foo.class)` to assertJ. I'll take a look at it (but initial look I had to assertJ migration was also heavy work, as stated in the PR comment)